naughtydog-decrypter/main.c

```diff
@@ -100,39 +257,40 @@ int main(int argc, char **argv)
 		return -1;
 	}
 
+	type = argv[2];
```

Comment:
just making sure the game option is uppercase. This might need `#include <ctype.h>`
```suggestion
	type = argv[2];
	type[1] = toupper(type[1]);
```